pyipmi/msgs/message.py

```diff
@@ -304,6 +304,11 @@ def __init__(self, name='picmg_identifier', value=None):
         UnsignedInt.__init__(self, name, 1, value)
 
 
+class EventMessageRevision(UnsignedInt):
+    def __init__(self, name='event_message_rev', value=None):
```

Comment:
Can the name have another value than 'event_message_rev'? Is it required that this is variable? If not we can drop that. And fill it fixed:

`UnsignedInt.__init__(self, 'event_message_rev', 1, value)`